domain/unitstate/errors/errors.go

```diff
@@ -9,4 +9,8 @@ const (
 	// UnitNotFound describes an error that occurs when
 	// the unit being operated on does not exist.
 	UnitNotFound = errors.ConstError("unit not found")
+
+	// UnitStateEmpty describes an error that occurs when
+	// the unit state is empty.
+	UnitStateEmpty = errors.ConstError("unit state empty")
```

Comment:
todo: update the comment to indicate that the error is used between the service and state, not an error callers into the domain will normally have to worry about.

Reply:
Removed this error entierly